src/Components/EmbeddedLogsExploration/EmbeddedLinkScene.tsx

```
@@ -41,6 +41,13 @@ export class EmbeddedLinkScene extends SceneObjectBase {
       return null;
     }
     const params = sceneUtils.getUrlState(indexScene);
+
+    // The link to breakdown cannot be created without labels
+    // Alternatively we could build a link to ServiceSelectionScene
+    if (labelsVarFilters.length === 0) {
+      return null;
```

Worth adding a warning logger to not silently fail.